src/components/ModalPayment/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -19,6 +19,9 @@ export type Props = {
   sellerName: string;
 };
 
+const lucHeroImage =
+  'https://storage.cloud.google.com/sendchinatownlove-assets/public/assets/light-up-chinatown/modal-payment-confirm-hero-LUC.png';
```

Comment:
Why not have the image in our assets?

Reply:
Mostly just trying to figure out whether or not we should bundle images or put it in the Storage Bucket instead. Might be over thinking it tbh